### ISSUE TITLE
Huge Boa Constrictor size issue

### ISSUE THREAD

**Issue by Uagr** [torilmud/issues#790]
A huge boa constrictor slowly slithers along.
look boa
This boa constictor is huge,  30 feet long and several feet in circumfrence.
A huge boa constrictor is a small-sized level 59 Snake.

A huge python is here, trying to hide.
look python
This is a huge python, must be 20 feet in length. It looks as if it could take
on even the largest troll.
A huge python is a small-sized level 55 Snake.

If these are small snakes god help me I don't want to find a huge one. Suggest updating size on them, also spelling. should be "circumference" on boa.

**Comment by vitaladsr** [comment]
Weird thought ... tho maybe it's something that can be added to templates .. but creatures that are oblong but not tall maybe should be no bash

**Comment by Uagr** [comment]
boa's & pythons used to be !bash, had to shield punch them.

**Comment by vitaladsr** [comment]
> boa's & pythons used to be !bash, had to shield punch them.

Yeah I thought so .. I remember they were pretty scary mofos back in the day .. aside from the old alligator 🐊 that mob still give me nightmares
